dbms/src/Storages/Transaction/TMTContext.cpp

```diff
@@ -36,6 +36,8 @@ extern const uint64_t DEFAULT_WAIT_INDEX_TIMEOUT_MS = 5 * 60 * 1000;
 
 const int64_t DEFAULT_WAIT_REGION_READY_TIMEOUT_SEC = 20 * 60;
 
+const int64_t DEFAULT_READ_INDEX_WORKER_TICK_MS = 10;
```

Comment:
also change the magic number 10 to be `DEFAULT_READ_INDEX_WORKER_TICK_MS` here https://github.com/pingcap/tiflash/blob/dcb78dbc89dc78939d9d0f2a42c7e0d5e10375e4/dbms/src/Storages/Transaction/TMTContext.cpp#L193